src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -159,7 +159,8 @@ public TestRunResult TestMultipleMutants(ITimeoutValueCalculator timeoutCalc, IR
                 testCases = null;
             }
 
-            var expectedTests = needAll ? DiscoverTests().Count : testCases.Count;
+            var numberTestCases = testCases == null ? 0 : testCases.Count;
```

Comment:
Why was this done?

Reply:
The line was [reported as a bug](https://sonarcloud.io/code?branch=v1.0&id=stryker-net&selected=stryker-net%3Asrc%2FStryker.Core%2FStryker.Core%2FTestRunners%2FVsTest%2FVsTestRunner.cs) by Sonar, rule S2259: _'testCases' is null on at least one execution path_.

I had to change the business logic a little, by defining that the number of test cases will be 0 if `testCases` is `null`.

Reply:
I don't think we can ever actually end up in this situation but because if we have no test cases there's nothing we can actually test anyway, but it is technically better 👌